live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/RequestFiltersIntegrationTest.java

```diff
@@ -37,13 +37,15 @@
         "alfresco.filter.aspect.allow[0]=cm:versionable", "alfresco.filter.aspect.allow[1]=cm:auditable",
         "alfresco.filter.type.deny[0]=cm:folder",
         "alfresco.filter.type.allow[0]=cm:content",
+        "alfresco.filter.path.allow[0]=5f355d16-f824-4173-bf4b-b1ec37ef5549", "alfresco.filter.path.allow[1]=93f7edf5-e4d8-4749-9b4c-e45097e2e19d",
+        "alfresco.filter.path.deny[0]=93f7edf5-e4d8-4749-9b4c-e45097e2e19e",
```

Comment:
Maybe edit these allow and deny UUIDs to be a bit more distinctive? We could use e.g. `11111111-1111-1111-1111-111111111111` as an easier to remember identifier?

Reply:
Brilliant suggestion, will do.